pkg/entrypoint/admissioncontroller/admissioncontroller_test.go

```diff
@@ -2,1280 +2,188 @@ package admissioncontroller
 
```

Comment:
Is the name really represents what it is doing? It is confusing. Maybe we should move it to `test/util` or smth and use it in the test where we need it? Or all the methods are local and we dont want to expose them?

Reply:
This is all local (likely to reuse in v4 though).